docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_QuartzNet.md

```diff
@@ -0,0 +1,27 @@
+# Convert PyTorch* QuartzNet to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_QuartzNet}
+
+[NeMo project](https://github.com/NVIDIA/NeMo) provides QuartzNet model.
+
+## Download the Pre-Trained QuartzNet Model
+
+To download pre-trained model please refer to [NeMo Developer Guide](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/).
```

Comment:
I moved by this link and didn't find any mention about this model. Can you make this link more concrete please?